src/backend/distributed/utils/statistics_collection.c

```diff
@@ -86,8 +100,138 @@ CollectBasicUsageStatistics(void)
 	escape_json(fields, unameData.machine);
 	appendStringInfoString(fields, "}");
 
-	return SendHttpPostJsonRequest(STATS_COLLECTION_HOST "/v1/usage_reports",
-								   fields->data, HTTP_TIMEOUT_SECONDS);
+	headers = curl_slist_append(headers, "Content-Type: application/json");
+	headers = curl_slist_append(headers, "charsets: utf-8");
+	success = SendHttpRequest(STATS_COLLECTION_HOST "/v1/usage_reports",
+							  headers, fields->data, HTTP_TIMEOUT_SECONDS,
+							  NULL /* no callbacks */);
+	curl_slist_free_all(headers);
+
+	return success;
+}
+
+
+/* CheckForUpdates queries Citus servers for newer releases of Citus. */
+void
+CheckForUpdates(void)
+{
+	SendHttpRequest(
+		STATS_COLLECTION_HOST "/v1/releases/latest?edition=" CITUS_EDITION,
```

Comment:
Even though we (at present) control the values of `CITUS_EDITION`, I think it would be best to URL-encode this. Check out libcurl's functions to do so.